documentation/modules/rn-2.6.adoc

```diff
@@ -141,6 +141,13 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-5_{context}"]
+=== Resolved issues 2.6.5
+
+.VMware Linux interface name changes during migration
+
+In earlier releases of {project-short}, during the migration of Rocky 8, CentOS 7.2+, and Ubuntu 22 virtual machines (VM) from VMware to {ocp} (OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```

Comment:
Please drop ` version 4.13` - as all OCP versions are affected.

Let's be specific about the verified Osses:

`resolved for static IPs in Rocky 8, Centos 7.2+, Ubuntu 22`